nservicebus/azure-service-bus/securing-connection-strings.md

```diff
@@ -6,56 +6,56 @@ tags:
 
-If this setting is enabled, the resulting `ReplyTo` header will look like this:
+When this setting is enabled, `ReplyTo` header will no longer contain raw connection string and will be structured as following
 
 ```no-highlight
-[queue name]@[namespacename]
+[queue name]@[alias]
 ```
 
 
 ## Configuration
 
-Mapping between a namespace name and the corresponding connection string, to perform cross namespace routing, is done as follows:
+To perform a cross-namespace routing, connection string has to be mapped to a corresponding alias. For example:
 
 snippet: namespace_routing_registration
 
-Note: All endpoints in a system need to be configured with the same namespace names and connection string information.
+Note: All endpoints in a system need to be configured with the same namespace aliases and connection string information.
 
-For a detailed explanation about all ways to configure namespace mappings for namespace routing and namespace partitioning, see [Multiple Namespaces Support](multiple-namespaces-support.md).
+For a detailed explanation on configuring namespace mappings for namespace routing and partitioning, see [Multiple Namespaces Support](multiple-namespaces-support.md).
 
 
 ## Backward compatibility
 
 include: asb-credential-warning
 
-Internally the transport (Version 7) uses the namespace name to refer to namespaces. Even when using the `ConnectionString(string connectionString)` method on the configuration API directly, as shown below, it will cause the transport to add a mapping between a namespace name `default` and the provided connection string internally.
+Internally the transport (Version 7 an above) uses namespace aliass to refer to namespaces. Even when using the `ConnectionString(string connectionString)` method on the configuration API directly, as shown below, it will cause the transport to add a mapping between a namespace alias `default` and the provided connection string internally.
 
-snippet: map_default_logical_name_to_connection_string
+snippet: map_default_logical_alias_to_connection_string
 
-Without enabling the `UseNamespaceNamesInsteadOfConnectionStrings()` behavior, the transport will ensure that all outbound headers are converted to the `queueName@connectionString` format before delivering the message. This ensures backward compatibility among endpoints of different versions.
+Without enabling the `UseNamespaceAliasInsteadOfConnectionStrings()` behavior, the transport will ensure that all outbound headers are converted to the `queueName@connectionString` format before delivering messages. This ensures backward compatibility among endpoints of different versions.
 
-By calling `UseNamespaceNameInsteadOfConnectionString()` the transport will change it's behavior, and send the namespace name instead.
+By calling `UseNamespaceAliasNameInsteadOfConnectionString()` the transport will change its behavior, and will not embed connections string in headers, using namespace aliases instead.
```

Comment:
How about:
By calling `UseNamespaceAliasNameInsteadOfConnectionString()` the transport will change its behavior. Instead of embedding connection strings in headers, namespace aliases will be used instead.